a10_neutron_lbaas/tests/db/neutron_ext/db/test_certificates.py

```diff
@@ -344,7 +344,7 @@ def test_update_certificate(self):
                                            'key_data': 'Key data',
                                            'intermediate_data': 'intermediate data',
                                            'password': 'password'}}
-        self.plugin.update_a10_certificate(ctx, cert['id'], certificate)
+        self.plugin.update_a10_certificate(ctx, cert['id'], **certificate)
```

Comment:
What's this change?

The test is invalid with this change; It's not passing data that looks like what the service accepts.